src/simulate/main.py

```diff
@@ -60,28 +61,52 @@ def main():
             sys.exit(1)
 
     files = os.listdir(json_files_dir)
+    session = requests.session()
     logging.info('list of json files in the directory %s', files)
+    event_count = 0
     for i in range(run_count):
         for file in files:
             if file.endswith(".json"):
                 path_to_file = json_files_dir + "/" + file
                 # Deserialize Json data
                 json_event = read_json(path_to_file)
                 if json_event is None:
+                    logging.error('Problem in reading Json event')
                     continue
                 # make needed changes to the json events
-                event_transformer = Transformer(json_event, config_yml_obj)
-                json_event_transformed = event_transformer.transformer()
+                json_event_transformed = get_transformed_event(config_yml_obj, json_event)
```

Comment:
This move doesn't seem necessary.  Why was it done?

Reply:
It's not needed, for readability sake it is taken to a separate method.